themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,38 +26,43 @@
     <label><?=$this->translate('Sort') ?></label>
     <div class="btn-group">
       <?php foreach ($this->sortOptions as $key => $sort): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) ?>" class="btn btn-default js-facet-sort<?php if ($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
+        <a href="<?=$this->escapeHtmlAttr($urlBase . '&facetpage=1&facetsort=' . urlencode($key) . '&contains=' . urlencode($contains)) ?>" class="btn btn-default js-facet-sort<?php if($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
       <?php endforeach; ?>
     </div>
+
+    <div class="btn-group facet-lightbox-filter hidden">
+      <form>
+        <fieldset>
+          <?php if ($this->results->getParams()->supportsFacetFiltering($this->facet) === true): ?>
+            <legend><?=$this->transEsc('Filter') ?></legend>
+            <input class="ajax_param" data-name="contains" class="form-control" type="text" value="<?=htmlspecialchars($this->contains)?>" aria-label="<?=$this->transEscAttr('search_terms')?>">
```

Comment:
class attribute is repeated

Reply:
This problem still exists -- there are two separate class attributes on this input, and they need to be merged.

Reply:
No, sorry, I was looking at the wrong version of the code. It's fine now! :-)